ui/components/app/edit-gas-popover/edit-gas-popover.component.js

```diff
@@ -62,6 +63,19 @@ export default function EditGasPopover({
   ] = useState(false);
 
   const minimumGasLimitDec = hexToDecimal(minimumGasLimit);
+  const updatedCustomGasSettings = useIncrementedGasFees(transaction);
```

Comment:
I think this is the root of the problem you're seeing. `useIncrementedGasFees` takes a transaction group, but you're passing in a single transaction here instead.

Reply:
Perhaps `useIncrementedGasFees` could be updated to take a single transaction? It doesn't seem to make any use of the transaction group - just the primary transaction.

Reply:
yes that's a far more sensible solution here... 🚀